### PR TITLE
Rustfmt Reapply

### PR DESCRIPTION
**Pull request by ChristopherDavenport**
It seems like we have a few features of rustfmt that havn't stayed fully current. 

This was current on nightly if that is what you are looking for.

### CODE REVIEW

File: src/iter/mod.rs [hunk 1]
```diff
@@ -561,12 +561,14 @@ pub trait ParallelIterator: Sized {
...
     fn find_last<P>(self, predicate: P) -> Option<Self::Item>
-        where P: Fn(&Self::Item) -> bool + Sync {
+        where P: Fn(&Self::Item) -> bool + Sync
+    {
         find_first_last::find_last(self, predicate)
     }
 
     #[doc(hidden)]
-    #[deprecated(note = "parallel `find` does not search in order -- use `find_any`, `find_first`, or `find_last`")]
+    #[deprecated(note = "parallel `find` does not search in order -- use `find_any`, \\
```
Comment (ChristopherDavenport) [inline src/iter/mod.rs]: I am uncertain if this is the appropriate way to handle splitting a Single string into a MultiLine String Split to lower 
line length which this exceeded.

Comment (nikomatsakis) [inline src/iter/mod.rs]: rustfmt is annoying like that.

**Comment by nikomatsakis** [comment]
Looks reasonably good to me, thanks @ChristopherDavenport.

**Comment by cuviper** [comment]
Should we apply this to rayon-core and rayon-demo too?

**Pull request merged**